### ISSUE TITLE
[Web Drop-in] challengeWindowSize type in CoreOptions.paymentMethodsConfiguration

### ISSUE THREAD

**Issue by michielmetcake** [Adyen/adyen-web#2537]
Using: 
`"@adyen/adyen-web": "5.57.0",`
Web Drop-in

When adding the `challengeWindowSize` prop as the [documentation](https://docs.adyen.com/online-payments/3d-secure/native-3ds2/web-drop-in/) describes, a type error is being returned.

Config:
```
paymentMethodsConfiguration: {
  // 3D Secure 2 window
  threeDS2: {
    challengeWindowSize: "05",
    },
}
```

The error is returned from `@adyen/adyen-web/dist/types/components/types.d.ts`
```Type '{ challengeWindowSize: string; }' has no properties in common with type 'Partial<UIElementProps>'```

No error is

**Comment by michielmetcake** [comment]
Introduced by [this](https://github.com/Adyen/adyen-web/commit/4f9a683e2a158ab5a1a2e0f6a9dfe4af9cedacb0) PR

**Comment by ribeiroguilherme** [comment]
Hi @michielmetcake ,

It should be fixed on 5.60.0. Cheers.
